### PR TITLE
Feature/angular app

### PR DESCRIPTION
**Pull request by dv-Gogna**
Integrate Web Chat in a app using angular framework

### CODE REVIEW

File: angular/src/app/app.component.ts [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+  title = 'Web Chat';
+
+  constructor(private renderer: Renderer2, private ngZone: NgZone) {}
+
+  isLoggedIn: boolean = false;
+  selectedPrivacyOption: string = 'disable';
+  selectedLauncherOption: string = 'showLauncher';
+  position: string = 'bottom-right';
+  selectedFullScreenOption: string = 'exitFullScreen';
+  unreadMessageCount: any = 0;
```
Comment (rohit369-hs) [inline angular/src/app/app.component.ts:26]: Can we have the type as [Number](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#the-primitives-string-number-and-boolean) ?

File: angular/src/app/app.component.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+          <button (click)="onLogin()">Login</button>
+          }
+        </div>
+      </div>
+      <div class="mt-2 flex justify-between">
+        <h4 class="text-lg py-2">Full Privacy</h4>
+        <div class="flex flex-col space-y-2 p-2">
+          <form>
+            <div class="inline-flex items-center mr-4">
+              <input
```
Comment (rohit369-hs) [inline angular/src/app/app.component.html:50]: [P3] - Can we use [formControl](https://angular.dev/guide/forms/reactive-forms) here for making it reactive?

File: angular/src/app/app.component.ts [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+        DOMAIN = "${environment.DOMAIN}",
+        LANGUAGE = "en";
+
+        // Do not modify the code below
+
+        // @NOTE: It's a good idea to set all widget options and information here.
+        // However, in the case where some information needs to be added later (async),
+        // use the updateHelpshiftConfig API https://developers.helpshift.com/web-chat/api/
+
+      window.helpshiftConfig = {
```
Comment (rohit369-hs) [inline angular/src/app/app.component.ts:84]: Since this object does not have anything specific to TS, can we move it to a utility file ?

Comment (dv-Gogna) [reply angular/src/app/app.component.ts:84]: Yes, we can do that 
but for now, we have made it simple basically to tell the clients where it should be placed.

File: angular/src/app/app.component.ts (hunk 1 above)
Comment (dv-Gogna) [reply angular/src/app/app.component.ts:26]: Yes will do this

File: angular/src/app/app.component.ts (hunk 3 above)
Comment (rohit369-hs) [reply angular/src/app/app.component.ts:84]: ok 👍

**Review by rohan-helpshift — APPROVED** [review]

**Review by rohit369-hs — APPROVED** [review]

**Pull request merged**